db/db_impl.cc

```diff
@@ -936,6 +985,7 @@ Status DBImpl::SetDBOptions(
           mutex_.Lock();
         }
         if (new_options.stats_persist_period_sec > 0) {
+          InitPersistStatsColumnFamily();
```

Comment:
Why do we need to call it again here while we already called it in DBImpl::Open()?

Reply:
this was added earlier when persist_to_disk was a mutable cf option, so we need to handle the case of after upgrading, user dynamically enabled on-disk persistent stats, which is not the case anymore, will be deleted